changelog.dd

```diff
@@ -7,7 +7,8 @@ $(BUGSTITLE Library Changes,
 
 $(LI $(RELATIVE_LINK2 TypeInfo.initializer, `TypeInfo.init` has been renamed to
     `TypeInfo.initializer`.))
-)
+$(LI $(RELATIVE_LINK2 AA.clear, Associative array `clear` method to remove all
+    elements has been added.))
 )
```

Comment:
Please use `RELATIVE_LINK aa-clear`, anchors shouldn't be camel cased.
How about "A `clear` method to remove all elements from an associative array has been added"?